GW2EIEvtcParser/EncounterLogic/Fractals/Deepstone/Deepstone.cs

```diff
@@ -0,0 +1,12 @@
+﻿using static GW2EIEvtcParser.EncounterLogic.EncounterCategory;
+
+namespace GW2EIEvtcParser.EncounterLogic;
+
+internal abstract class Deepstone : FractalLogic
+{
+    public Deepstone(int triggerID) : base(triggerID)
+    {
+        EncounterCategoryInformation.SubCategory = SubFightCategory.Deepstone;
+        EncounterID |= 0;
```

Comment:
Set to zero ^^ this operation will conserve the fractal mask. Same for Mai Trin.